File: Assignment14/program5.c
/*
5.Write a program which accept number from user and count frequency of such a
digits which are less than 6.

Input  :    2395
Output :    3

*/
#include <stdio.h>

int Digits(int iValue)
{
    int iDigit = 0;
    int iFreq = 0;

        while(iValue != 0)
        {
            iDigit = iValue % 10;
            iValue = iValue / 10;

            if(iDigit <= 6)
            {
                iFreq = iFreq + 1;
            }
        }
        

        return iFreq;


}

int main()
{
    int iValue = 0,iRet = 0;

    printf("Enter the Number : ");
    scanf("%d",&iValue);

    iRet = Digits(iValue);
    printf("Frequency The Number Less than 6  is : %d\n",iRet);
    
    return 0;
}

File: Assignment16/program5.java

// 5. Write a program to count the number of digits in a given number.

package Assignment16;

import java.util.Scanner;

public class program5 
{
    public int CountDigits(int iValue)
    {
        int iDigit = 0,iCount = 0;

        while(iValue != 0)
        {
            iDigit = iValue % 10;
            iCount = iCount + 1;
            iValue = iValue / 10;
        }
        return iCount;
    }

    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        program5 pobj = new program5();

        int iValue = 0,iRet = 0;
        

        System.out.println("Enter the Number : ");
        iValue = sobj.nextInt();

        iRet = pobj.CountDigits(iValue);
        System.out.println("the Number of Digits in Number is : "+iRet);

        sobj.close();
    }    
}
